### DISCUSSION TITLE
Bug? On Vote, I can downvote an item to steal some one elses vote

### DISCUSSION THREAD

**Discussion by joshuagilroysainsburys** [microsoft/vsts-extension-retrospectives#477] [General]
Hi,

Noticed in the tool that during voting, I can downvote an item I have not voted on to "steal" others votes already on item.
ie,

item 6 has 3 upvotes, but none are mine. If I hit downvote, the item goes to 2 upvotes and I get an extra vote to use and can use on any other item, even if I have used all my votes already.
If I have 5 max votes, I can use this technique to put many more votes across the board on different items.

Think its related to this area - https://github.com/microsoft/vsts-extension-retrospectives/blob/7e95c2db7f0a268f8f06bfb2d6d6997186392e78/RetrospectiveExtension.Frontend/dal/itemDataService.tsx#L249

Surely should be checking if feedbackItem.voteCollection[userId] > 0 ? Either that or the UserID is not working as intended. As above this affects items that have not been voted on before so an odd thing, perhaps just on anonymous user setting?

**Comment by vegar** [comment]
yep. This is still an issue apparently. First time I stole a vote, I didn't undestand what happened. Second time, I was a little surpriced. Third time I felt a lttle bit evil.  😈

**Comment by dphanson63** [comment]
Hasn't been an issue for awhile.
